src/test/java/SoloTech/StockFlow/order/OrderServiceTest.java

```diff
@@ -138,122 +139,124 @@ void createOrderTest() {
-                "S12345",      // storeId
-                "P56789",      // productId
-                "STK98765",    // stockId
-                2L,            // quantity
-                20000L,        // amount
-                "CARD"         // paymentMethod
-        );
-
+        // given
+        String orderId = "ORD1001";
         Order existingOrder = Order.builder()
+                .id(1L)
                 .orderId(orderId)
-                .storeId("storeOld")
-                .productId("productOld")
-                .stockId("stockOld")
+                .storeId("S001")
+                .productId("P001")
+                .stockId("ST001")
+                .quantity(2L)
+                .build();
+
+        OrderDto updateDto = OrderDto.builder()
                 .quantity(5L)
                 .build();
+
         Order updatedOrder = Order.builder()
+                .id(1L)
                 .orderId(orderId)
-                .storeId(dto.getStoreId())
-                .productId(dto.getProductId())
-                .stockId(dto.getStockId())
-                .quantity(dto.getQuantity())
+                .storeId("S001")
+                .productId("P001")
+                .stockId("ST001")
+                .quantity(5L)
                 .build();
-        String cacheKey = ORDER_KEY_PREFIX + orderId;
-        String message = "Updated order-" + cacheKey;
 
         when(orderRepository.findByOrderId(orderId)).thenReturn(Optional.of(existingOrder));
-        // ObjectMapper의 updateValue를 모킹하여 예외 없이 작동하도록 설정
-        doAnswer(invocation -> {
-            Order target = invocation.getArgument(0); // 첫 번째 매개변수
-            OrderDto source = invocation.getArgument(1); // 두 번째 매개변수
-            target.setStoreId(source.getStoreId());
-            target.setProductId(source.getProductId());
-            target.setStockId(source.getStockId());
-            target.setQuantity(source.getQuantity());
-            return null; // updateValue는 void이므로 null 반환
-        }).when(mapper).updateValue(existingOrder, dto);
-
+        doNothing().when(mapper).updateValue(existingOrder, updateDto);
         when(orderRepository.save(existingOrder)).thenReturn(updatedOrder);
-        when(localCache.getIfPresent(cacheKey)).thenReturn(null);
 
-        // When
-        Order result = orderService.updateOrder(orderId, dto);
+        // when
+        Order result = orderService.updateOrder(orderId, updateDto);
 
-        // Then
-        assertEquals(updatedOrder, result);
-        verify(localCache).put(cacheKey, updatedOrder);
-        verify(cachePublisher).publish("cache-sync", message);
+        // then
+        assertNotNull(result);
+        assertEquals(5L, result.getQuantity());
+        verify(orderRepository).findByOrderId(orderId);
+        verify(mapper).updateValue(existingOrder, updateDto);
         verify(orderRepository).save(existingOrder);
-        verify(mapper).updateValue(existingOrder, dto);
+    }
+
+    @Test
```

Comment:
```suggestion
    @Test
    @DisplayName("주문이 없는 경우~~~~")
```

한글로도  표현하면 더 가독성이 높을거 같아요!